src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyObjectGenerator.cs

```diff
@@ -555,7 +555,6 @@ public void RestoreBindings()
 
 					public void ApplyCompiledBindings()
 					{{
```

Comment:
maybe a comment here (eg 'this is left for binary compatibility but...')

Reply:
Will do in next PR.